api/proto/dop/qa/unittest/unittest.proto

```diff
@@ -0,0 +1,168 @@
+syntax = "proto3";
+
+package erda.dop.qa.unittest;
+option go_package = "github.com/erda-project/erda-proto-go/dop/qa/unittest/pb";
+import "google/api/annotations.proto";
+import "common/openapi.proto";
+import "google/protobuf/timestamp.proto";
+
+service UnitTestService {
+  option(erda.common.openapi_service) = {
+    service : "dop",
+    auth : {
+      check_login : true,
+      check_token: true,
+    }
+  };
+  rpc Callback(TestCallBackRequest) returns (TestCallBackResponse) {
+    option (google.api.http) = {
+      post : "/api/qa/actions/test-callback",
+    };
+    option (erda.common.openapi) = {
+      path : "/api/qa/actions/test-callback",
+    };
+  }
+  rpc GetTestTypes(TestTypeRequest) returns (TestTypeResponse) {
+    option (google.api.http) = {
+      get : "/api/qa/actions/all-test-type",
+    };
+    option (erda.common.openapi) = {
+      path : "/api/qa/actions/all-test-type",
+    };
+  }
+  rpc ListRecords(TestRecordPagingRequest) returns (TestRecordPagingResponse) {
+    option (google.api.http) = {
+      get : "/api/qa/actions/test-list",
+    };
+    option (erda.common.openapi) = {
+      path : "/api/qa/actions/test-list",
+    };
+  }
+  rpc GetRecord(TestRecordGetRequest) returns (TestRecordGetResponse) {
+    option (google.api.http) = {
+      get : "/api/qa/test/{id}",
+    };
+    option (erda.common.openapi) = {
+      path : "/api/qa/test/{id}",
+    };
+  }
+}
+
+message TestRecordPagingRequest {
+  int64 pageNo = 1;
+  int64 pageSize = 2;
+  uint64  applicationId =3;
```

Comment:
bad format

Reply:
i had `make format`, there are what it looks like after formatting
![image](https://user-images.githubusercontent.com/30427474/167242871-9bac1a45-2313-44af-90e6-f67ca6f4cf1e.png)
other protos are the same format

Reply:
adjusted, make format by protobuf plugin